json/src/test/java/io/airlift/json/Person.java

```diff
@@ -22,24 +22,46 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.Optional;
 
 import static com.google.common.base.MoreObjects.toStringHelper;
 import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
 
 public class Person
 {
     private String name;
     private boolean rocks;
+    private Optional<String> lastName;
 
     public static void validatePersonJsonCodec(JsonCodec<Person> jsonCodec)
     {
         Person expected = new Person().setName("dain").setRocks(true);
 
         String json = jsonCodec.toJson(expected);
+        assertEquals(json.indexOf("lastName"), -1);
```

Comment:
Add a comment before each section to make it easier to know what is being tested
```java
// create object with null lastName

// create object with missing lastName

// create object with present lastName